juniper_official/Solutions/Icmp-outlier/icmp.rule

```diff
@@ -0,0 +1,252 @@
+/*
+ * Sends ICMP probes to user specified destination host and notifies anomalies 
+ * when round trip time average response is above static or dynamic thresholds.
+ * And also detects outlier across the devices in device group ML algorithms.
+ * Three inputs control detection
+ * 
+ *   1) host_var, is a destination IP or host name where ICMP probes send 
+ *      periodically.
+ *   2) count_var, is a ICMP ping count. By default count is set to 1. 
+ *   3) rtt-threshold-var, is the threshold that causes the rule to report an
+ *      anomaly. By default it is 1000 micro seconds. This rule will set a
+ *      dashboard color to red when ICMP response time is greater than static
+ *      threshold 'rtt-threshold-var'.
+ */
+ iceberg {
+    topic protocol.icmp {
+        description "Sends ICMP probes to destination host and notify anomalies";
+        synopsis "ICMP response analyzer";
+        rule check-icmp-statistics {
+            /*
+             * Sends ICMP probes to destination host. Notifies via the dashboard
+             * colors when the host is not reachable or response time is above
+             * threshold {{rtt-threshold-var}} to red. (red). The color is set
+             * to yellow if response time is above dynamic threshold. Otherwise
+             * the color is set to green.
+             * 
+             * Use host as key for rule.
+             */
+             keys host;
+             /*
+             * Sensor configuration to get data from network devices.
+             */
+             sensor icmp {
+                synopsis "iAgent sensor definition";
+                description "Pin rpc to collect data from network device";
```

Comment:
Rephrase "pin" to "Ping" and "rpc" to "RPC".

Reply:
Modified